### PR TITLE
Add job bosh-openstack-cpi-release-acceptance-test

### PR DESCRIPTION
**Pull request by animationzl**

### CODE REVIEW

**Comment by animationzl** [comment]
Will move these code into cloudfoundry-incubator/bosh-openstack-cpi-release repo after test passed

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -340,3 +340,13 @@
     nodeset: ubuntu-xenial-otc
     secrets:
       - opentelekomcloud_credentials
+
+- job:
+    name: bosh-openstack-cpi-release-acceptance-test
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: please add openstack version `newton` in job name, like: bosh-openstack-cpi-release-acceptance-test-newton

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -340,3 +340,13 @@
...
+    parent: init-test
+    description: |
+      This job validation tests of bosh-openstack-cpi-release aganist devstack
```
Comment (kiwik) [inline zuul.d/jobs.yaml]: s/aganist devstack/aganist OpenStack Newton release/

File: playbooks/bosh-openstack-cpi-release-acceptance-test/run.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        rm bosh-openstack-kvm-ubuntu-trusty-go_agent.tgz
+        sed -i 's/kvm/qemu/g' stemcell.MF
+        tar -zcvf bosh-openstack-qemu-ubuntu-trusty-go_agent.tgz *
+        mv bosh-openstack-qemu-ubuntu-trusty-go_agent.tgz ../
+        popd
+
+        # Run test
+        bundle install
+        ./validate -s bosh-openstack-qemu-ubuntu-trusty-go_agent.tgz -c validator.yml -r bosh-openstack-cpi.tgz --tag cpi_api
+
```
Comment (kiwik) [inline playbooks/bosh-openstack-cpi-release-acceptance-test/run.yaml]: popd

**Pull request merged**